src/index.tsx

```diff
@@ -38,6 +38,8 @@ MapboxGL.setAccessToken(MAPBOX_API_TOKEN);
 
 trackAppState();
 
+LogBox.ignoreLogs(['new NativeEventEmitter']);
```

Comment:
Mm, I know is a warning, but it is possible to fix it? or is this a temp solution?

Reply:
It is at least not code we use, but libraries that use some deprecated stuff. Hopefully the libraries should fix it themselves. It might be possible to contribute with PRs to those libraries if we wan't to speed up the process.